admin/admin-service/src/main/java/io/mosip/admin/controller/AdminController.java

```diff
@@ -0,0 +1,34 @@
+package io.mosip.admin.controller;
```

Comment:
I see the packet structure used in the admin is different from below, can you get a confirmation from Anusha on this?

https://github.com/mosip/admin-services/blob/1.1.5.3/admin/admin-service/src/main/java/io/mosip/admin/bulkdataupload/contoller/BulkDataUploadController.java

Reply:
previously they have created wrong pkg structure so i have modified

Reply:
Resolved